### ISSUE TITLE
Flowdroid with Cordova application (Plugins)

### ISSUE THREAD

**Issue by arvind4141** [secure-software-engineering/soot-infoflow-android#186]
Hello, I'm currently trying to build a cfg of a Cordova application which utilises external plugins.
The existing cfg has an entry point from "MainActivity" which the cfg is propogated from and produces results the same as with any other Native application.

However, how do I allow the cfg to build edges into classes that do not have an entry point? 
As you can see in the below example, the plugin "Sms" does not have an entry point. 
![capture](https://user-images.githubusercontent.com/22865857/33720579-563e6c22-db9f-11e7-8620-4596374afaf5.PNG)
If it is not possible to build a cfg into classes that do not have an entry point, how do I tell Flowdroid that  I would like a cfg to be built from this "single class" and then perform possible "taint analysis" within this class itself. Is this even possible?

In short to avoid confusion - these are the things that I would like to achieve.
1. Allow CFG generation to include this specific class (If this is not possible its okay)
2. Build a CFG based on this class within the android (I believe this can be done on Soot, correct me if I'm wrong thank you)
3. Perform Taint Analysis within this class (Is this possible?)

Thank you for the reply, I really appreciate it.

**Comment by arvind4141** [comment]
@StevenArzt Some help would greatly be appreciated here! Thank you!
